src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,17 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Machine truck = new Truck();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
 
+        Machine[] machineArray = new Machine[] {truck, bulldozer, excavator};
```

Comment:
In such a case, there is no need to store each object in a separate variable. You can put new objects directly inside an array like this: Machine[] {new Truck(), ...}

Reply:
Yea, rewrite it in that way:
```
Machine[] machineArray = new Machine[] {new Truck(), new Bulldozer(), new Excavator()};
```
Then you will realize, that 4-7 lines are redundant.